chat/conf/locale

```diff
@@ -0,0 +1 @@
+../translations
```

Comment:
This should be in `chat/locale`, actually a sibling of `translations`

Reply:
I thought edx i18n tools use this `chat/conf/locale` directory.
There's another symlink at the root directory with the name `locale` which also goes to translations. I think that's the one I have to move to `/chat/.`